### ISSUE TITLE
Typo in metric name 'cert_subject_alternative_name' of remote.http check

### ISSUE THREAD

**Issue by itzg** [rackerlabs/docs-cloud-monitoring#48]
## What do you want us to know about?  
*(Check all that apply.)*

**Report a documentation issue**
- [X] Bug (misspellings, grammatical problems, broken link, format issues)
- [ ] Missing information
- [ ] Incorrect information
- [X] Feedback or suggestions
- [ ] Other

**Product**
- [ ] Product bug 
- [ ] Product feedback

**Note:** For product-related issues, you can also submit an issue from the [Rackspace Community Forum](https://community.rackspace.com/) website. 

**Tell us more about your issue**

The metric name highlighted here is missing an "s". It should be `cert_subject_alternative_names`.

![image](https://user-images.githubusercontent.com/988985/33456948-9928f7be-d5e6-11e7-9623-febea8a3c63b.png)

I'm not sure where this should go, but I would recommend providing an example alarm snippet for that metric that contains:

```
if (metric['cert_subject_alternative_names'] nregex '.*example.com.*') {
  return new AlarmStatus(CRITICAL, 'Missing expected SAN');
}
```

where the customer replaces "example.com" with an expected hostname on the certificate's SAN list.

**Comment by cyrichardson** [comment]
Thank you! I will get these changes done this week and ask for your review, @itzg.

**Comment by cyrichardson** [comment]
Change in PR #50. Change made in remote.http section. Do they need to be made in remote.smtp-banner as well? There is an occurrence of cert_subject_alternative_name there.

@itzg Please review content change in PR.

**Comment by itzg** [comment]
Sorry, I had missed your question at first. Good catch -- it does look like the plural is the correct form in all check types.

**Comment by cyrichardson** [comment]
@itzg Thanks! I'll fix the 2nd occurrence too and make it part of the same PR. Really appreciate your reporting and review of the PR!

**Comment by cyrichardson** [comment]
Closed by #50
